src/test/resources/report_seed_data_test.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
Why are there two JSON files? I'd expect only one

Reply:
One for the test, and one for the implementation - but will remove the test one.